kzz7yh-f81481/cod-ve09v2_kzz7yh-f81481.xml

```diff
@@ -357,9 +361,9 @@
                   <lb ed="#V" n="69"/>tam eternam de condigno. 
    </p>
                <p xml:id="kzz7yh-f81481-d1e648">
-                  <lb ed="#V" n="70"/>Contra glosa super illud ad Rom. 6 gratia 
-      deivi<lb ed="#V" n="71" break="no"/>ta eterna: confitendum est: ideo gratiam vi 
-      <lb ed="#V" n="72"/>tam eternam vocari: quia his meritis redditur que 
+                  <lb ed="#V" n="70"/>Contra glosa super illud ad <ref>Rom. 6.</ref> gratia 
```

Comment:
```suggestion
                  <lb ed="#V" n="70"/>Contra glosa super illud <ref>ad Rom. 6.</ref> gratia 
```

Reply:
These are all St. Paul's Letters "to the _____" so I think the "ad" should be included in the ref.